test/upgrade/helpers.bash

```diff
@@ -9,3 +9,11 @@ setup() {
 teardown() {
     :
 }
+
+# skip a test when the given version is older than the currently tested one
+skip_if_version_older() {
+    # use ${PODMAN_UPGRADE_FROM:1} to trim the leading "v"
+    if printf '%s\n%s\n' "${PODMAN_UPGRADE_FROM:1}" "$1" | sort --check=quiet --version-sort; then
+        skip "version $PODMAN_UPGRADE_FROM is to old $2"
```

Comment:
Suggestion: `skip "${2-test is only meaningful when upgrading from $1 or later}"`. This lets the caller optionally give a reason, but if there isn't one, we still give a helpful message.